### PR TITLE
fix bug with mempool explorer url

### LINKED ISSUE #384: With Mempool exploer the "See in explorer" button does not follow correctly
It try to open this URL 

https://liquid.network/#blinded=229228849,144c654344aa716d6f3abcc1ca90e5641e4e2a7f633bc09fe3baf64585819a49,ef7d56ee5361616c3070acd11314384c80d566506435a952559a7a5ef6b59f93,9adbba53ca4dae4807f2e6d5fe64d567182bdefd56094897801554840eacd9a9

missing /<network>/tx/<txid>

https://www.loom.com/share/29bca706db5b48a2b1b7c7c74a598d07

### PR DESCRIPTION
**Pull request by bordalix**
It closes #384 

@tiero please review

### CODE REVIEW

File: src/domain/app.ts [hunk 1]
```diff
@@ -45,6 +45,6 @@ export const MempoolExplorerURLs: ExplorerURLs = {
 
 export const MempoolTestnetExplorerURLs: ExplorerURLs = {
   type: 'Mempool',
-  electrsURL: 'https://liquid.network/liquidtestnet',
-  esploraURL: 'https://liquid.network/liquidtestnet/api',
+  electrsURL: 'https://liquid.network/testnet',
+  esploraURL: 'https://liquid.network/testnet/api',
```
Comment (tiero) [inline src/domain/app.ts:49]:
```suggestion
  electrsURL: 'https://liquid.network/liquidtestnet/api',
  esploraURL: 'https://liquid.network/testnet',
```

It seems to me the elctrs backend API can be found at 

https://liquid.network/liquidtestnet/api/blocks

and frontend at

https://liquid.network/testnet/blocks

File: src/domain/app.ts [hunk 2]
```diff
@@ -45,6 +45,6 @@ export const MempoolExplorerURLs: ExplorerURLs = {
 
 export const MempoolTestnetExplorerURLs: ExplorerURLs = {
   type: 'Mempool',
-  electrsURL: 'https://liquid.network/liquidtestnet',
+  electrsURL: 'https://liquid.network/testnet',
```
Comment (tiero) [inline src/domain/app.ts:48]: I think is opposite. electrs is bakcend and esploraURL is frontend?

**Comment by bordalix** [comment]
It should be
```typescript
  electrsURL: 'https://liquid.network/testnet',
  esploraURL: 'https://liquid.network/liquidtestnet/api',
```
to be consistent with the other providers (esplora = 'api')

changed by 7437a6a

File: src/domain/app.ts (hunk 2 above)
Comment (bordalix) [reply src/domain/app.ts:48]: On the other providers is not:

```typescript
export const BlockstreamExplorerURLs: ExplorerURLs = {
  type: 'Blockstream',
  electrsURL: 'https://blockstream.info/liquid',
  esploraURL: 'https://blockstream.info/liquid/api',
};

export const BlockstreamTestnetExplorerURLs: ExplorerURLs = {
  type: 'Blockstream',
  electrsURL: 'https://blockstream.info/liquidtestnet',
  esploraURL: 'https://blockstream.info/liquidtestnet/api',
};

export const NigiriDefaultExplorerURLs: ExplorerURLs = {
  type: 'Nigiri',
  electrsURL: 'http://localhost:5001',
  esploraURL: 'http://localhost:3001',
};

export const MempoolExplorerURLs: ExplorerURLs = {
  type: 'Mempool',
  electrsURL: 'https://liquid.network',
  esploraURL: 'https://liquid.network/api',
};

export const MempoolTestnetExplorerURLs: ExplorerURLs = {
  type: 'Mempool',
  electrsURL: 'https://liquid.network/testnet',
  esploraURL: 'https://liquid.network/liquidtestnet/api',
};
```

**Pull request merged**